02-wrangling.Rmd

```diff
@@ -175,7 +188,7 @@ lang_mother_tidy
 > *to read clearly. When doing this, it is important to end the line with a comma `,` so that R*
```

Comment:
We should state the number of recommended characters here (80 I think?) and there is an extra `*` at the end of this line I think.